packages/heartwood-components/components/04-images-and-icons/01-avatar/avatar.config.js

```diff
@@ -15,35 +20,40 @@ module.exports = {
 			context: {
 				image
 			}
-		}, {
```

Comment:
Just wanna be sure `prettier`/`eslint` is configured properly here, I thought our ruleset kept opening braces on the same line if chained (could be wrong about that though)

Reply:
@gcpantazis looks like the prettier settings are the same as other projects. The vscode settings were not part of the project though, I have added the settings for format on save to the other PR I opened.